data/functions.sql

```diff
@@ -178,6 +178,10 @@ BEGIN
       WHEN amenity_val  = 'ski_rental'       THEN LEAST(zoom + 1.27, 17)
       WHEN amenity_val  = 'ski_school'       THEN LEAST(zoom + 2.30, 15)
       WHEN man_made_val = 'snow_cannon'      THEN LEAST(zoom + 4.90, 18)
+      WHEN leisure_val  = 'sports_centre'    THEN LEAST(zoom + 3.98, 17)
+      WHEN leisure_val  = 'fitness_centre'   THEN LEAST(zoom + 3.98, 17)
+      WHEN leisure_val  = 'fitness_station'  THEN LEAST(zoom + 3.98, 17)
+      WHEN amenity_val  = 'gym'              THEN LEAST(zoom + 3.98, 17)
```

Comment:
Yes, `amenity=gym` should become `kind=gym` when coalesced in the query, and the transform should normalise that to `kind=fitness` [on this line](https://github.com/mapzen/TileStache/pull/110/files#diff-52de7615e270521ccb11d3a0fe443d33R2713).